addons/utm/models/utm.py

```diff
@@ -34,6 +34,13 @@ class UtmCampaign(models.Model):
     is_website = fields.Boolean(default=False, help="Allows us to filter relevant Campaign")
     color = fields.Integer(string='Color Index')
 
+    # After merged with other utm campaign, unneeded one should be deactived, and it should refer to 
+    # the new campaign to transfer statistics collected after merge to the new one.
```

Comment:
Remove those comments as they're now included in the `reference_utm_campaign_id` help attribute.